### PR TITLE
Create stories from analyzers

### PR DESCRIPTION
**Pull request by berggren**
This PR adds the ability to create and author a Story from analyzers.

```
        view = self.sketch.add_view(
            view_name='Story test', analyzer_name=self.NAME,
            query_string='foobar')
        story = self.sketch.add_story(title='Story from analyzer')
        story.add_text('## This is a markdown title')
        story.add_view(view)
        story.add_text('This is another paragraph')
```

### CODE REVIEW

File: timesketch/lib/analyzers/interface.py [hunk 1]
```diff
@@ -351,6 +353,26 @@ def add_view(self, view_name, analyzer_name, query_string=None,
...
+        Args:
+            title: The name of the view.
+
+        Raises:
+            ValueError: If both query_string an query_dsl are missing.
+
+        Returns:
+            An instance of a Story object.
+        """
+
```
Comment (kiddinn) [inline timesketch/lib/analyzers/interface.py]: no need for this extra space/line

File: timesketch/lib/analyzers/interface.py [hunk 2]
```diff
@@ -351,6 +353,26 @@ def add_view(self, view_name, analyzer_name, query_string=None,
...
+        story = SQLStory.get_or_create(
+            title=title, content=json.dumps([]), sketch=self.sql_sketch,
```
Comment (kiddinn) [inline timesketch/lib/analyzers/interface.py]: I don't see a reason to call `json.dumps([])` here, that is a constant... calling `json.dumps([])` will produce the string: `'[]'`, and therefore just have `content='[]', ` should suffice

File: timesketch/lib/analyzers/interface.py (hunk 1 above)
Comment (berggren) [reply timesketch/lib/analyzers/interface.py]: done

File: timesketch/lib/analyzers/interface.py (hunk 2 above)
Comment (berggren) [reply timesketch/lib/analyzers/interface.py]: good catch, fixed.

**Comment by berggren** [comment]
PTAL @kiddinn

**Pull request merged**